### PR TITLE
Convert java/javapayload/index.html to README.md

### PR DESCRIPTION
**Pull request by wvu-r7**
Converted hastily with `pandoc` and edited minimally:

```
pandoc -f html -t gfm index.html -o README.md
```

How it looks in the repo: https://github.com/rapid7/metasploit-payloads/tree/feature/java/java/javapayload.

https://github.com/rapid7/metasploit-framework/pull/11564

### CODE REVIEW

File: java/javapayload/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+# JavaPayload4Metasploit - Single payload loader class to be used in the Metasploit project
+
+*© 2010 Michael 'mihi' Schierl, `<schierlm at users dot sourceforge dot
+net>`*
+
+## Introduction
+
+The [JavaPayload](http://schierlm.users.sourceforge.net/JavaPayload/)s
```
Comment (bcoles) [inline java/javapayload/README.md]: Not sure if intentional.

```suggestion
The [JavaPayload](http://schierlm.users.sourceforge.net/JavaPayload/)
```

File: java/javapayload/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+contain useful payloads written in pure Java. But they assume that the
+attacker has a Java VM on his machine, as the the builders and stage
```
Comment (bcoles) [inline java/javapayload/README.md]:
```suggestion
attacker has a Java VM on their machine, as the builders and stage
```

File: java/javapayload/README.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+## Supported properties (and their default values)
+
+### `Spawn`(`=0`)
+
+The number of java processes that should be spawned. `0` will run the
+payload inside the original process, `1` will spawn once (to continue
+running when the original process terminates), and `2` will spawn twice
+(on certain popular operating systems it is impossible to obtain parent
+process informaion if the parent process has already died).
```
Comment (bcoles) [inline java/javapayload/README.md]:
```suggestion
process information if the parent process has already died).
```

File: java/javapayload/README.md (hunk 1 above)
Comment (wvu-r7) [reply java/javapayload/README.md]: I believe it was intentional.

**Pull request merged**